.github/workflows/mypy.yml

```
@@ -48,7 +48,7 @@ jobs:
         os_python:
           - [ubuntu-latest, '3.12']
           - [windows-2019, '3.11']
```

As long as we are changing this, maybe do windows too?
```suggestion
          - [windows-latest, '3.11']
```